### DISCUSSION TITLE
text-size inherit ?

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [tailwindlabs/tailwindcss#10691] [Help] [answered]
Is there no inherit for text size? `text-inherit` inherits color

**Comment by wongjn** [comment, marked as answer]
There is. You could use an arbitrary value with a type cast: `text-[length:inherit]`.

**Reply by LamprosPitsillos** [reply]
Thanks! Do you mind explaining why it needed `length ` because i tried several arbitrary values with no luck?

**Reply by wongjn** [reply]
`text-` prefix is used by the text color and text size utilities so the `length:` lets Tailwind know which of these utilities the arbitrary value is for.

**Comment by stijns96** [comment]
Hi @wongjn,

Looks like it is not working in TW4. Any solution for this?

**Reply by wongjn** [reply]
Seems to be working in [this Tailwind Play](https://play.tailwindcss.com/84KWNJo8Is) which would suggest if it is not working for you, it could be something particular to your project.

**Reply by stijns96** [reply]
Don't ask me why, but it is working now... Thanks for the very quick response!
